js/mexc.js

```diff
@@ -627,8 +625,8 @@ module.exports = class mexc extends Exchange {
                 'strike': undefined,
                 'optionType': undefined,
                 'precision': {
-                    'amount': this.safeNumber (market, 'volUnit'),
-                    'price': this.safeNumber (market, 'priceUnit'),
+                    'amount': this.parseNumber (this.parsePrecision (this.safeString (market, 'volScale'))),
```

Comment:
@ttodua you have you tested this with both the spot and and the swap defaultTypes right ?